src/components/ListOfTipsAndComments.vue

```diff
@@ -167,13 +167,14 @@ export default {
     },
     reloadData() {
       this.showLoading = true;
-      Backend.getAllComments()
-        .then((allComments) => {
+      Backend.getUserComments(this.address)
+        .then((userComments) => {
           this.showLoading = false;
           this.error = false;
-          this.comments = allComments.filter(
-            (comment) => comment.author === this.address,
-          );
+          if (userComments.length) {
```

Comment:
This condition looks unnecessary. I don't think that it is bad to call `sort` of an empty array. Also, it would be easier to read this without this condition. In general, js is too slow for making such optimizations.